src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using Microsoft.AspNetCore.Hosting;
+
+namespace WebJobs.Script.WebHost.Core
```

Comment:
Why the new namespace? All the other "core" types are in the root namespace "Microsoft.Azure.WebJobs.Script.WebHost"

Reply:
This was leftover from the initial migration. Should have been fixed